### DISCUSSION TITLE
Enable predictive back gesture support (android 14+)

### DISCUSSION THREAD

**Discussion by rayyangh** [libre-tube/LibreTube#6862] [Ideas]
### Describe your suggested feature

its such a minor enhancement and this should be labeled low priority.

in AndroidManifest.xml, in the tag, set the android:enableOnBackInvokedCallback flag to true.
```
<application
    ...
    android:enableOnBackInvokedCallback="true"
    ... >
...
</application>
```

### Other details

link to android developer page explaining this feature: https://developer.android.com/guide/navigation/custom-back/predictive-back-gesture

### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
That's already set to true, but the app would need a lot of additional modifications to properly support predictive back gestures.

**Comment by Bnyro** [comment]
Added in https://github.com/libre-tube/LibreTube/pull/6863
